lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -21,12 +21,25 @@ import '../constants/typedefs.dart';
 import '../delegates/asset_picker_text_delegate.dart';
 import '../internals/singleton.dart';
 import '../models/path_wrapper.dart';
+import '../models/special_item.dart';
 import '../provider/asset_picker_provider.dart';
 import '../widget/asset_picker.dart';
 import '../widget/asset_picker_app_bar.dart';
 import '../widget/asset_picker_viewer.dart';
 import '../widget/builder/asset_entity_grid_item_builder.dart';
 
+/// Class which contains non-null special item widget and its position which derived from the [SpecialItem]
+/// 包含非空自定义item，并指定其位置。
+class SpecialItemResult {
```

Comment:
Move back to the spacial item model and make it const

Reply:
updated to SpecialItemModel and made it const.